apps/dashboard/src/main/java/com/akto/action/ApiCollectionsAction.java

```diff
@@ -373,6 +367,16 @@ public String fetchTimersInfo(){
         return Action.ERROR.toUpperCase();
     }
 
+    public String fetchCustomerEndpoints(){
+        try {
+            this.usageEndpoints = UsageMetricCalculator.calculateActiveEndpoints(0);
```

Comment:
Hi @Ark2307 - we shouldn't recalculate here - we should simply fetch last usage data from database